packages/widgetbook/lib/src/widgetbook.dart

```diff
@@ -101,6 +101,7 @@ class _WidgetbookState extends State<Widgetbook> {
     );
 
     router = AppRouter(
+      initialRoute: Uri.base.fragment,
```

Comment:
This will only work on web, right?

Reply:
Non-web should not change behaviour (same as typing a bad URL, it will send you back to the root path). 

The docs say `Uri.base` refers to the current working directory when not on web.

My assumptions were:

(a) most people are using Widgetbook in the browser,
(b) remembering routes in an app requires storing state on disk in some fashion (and possibly implementing deep-linking), and is a significantly more complex solution,

Happy to have these challenged though :-)

Reply:
I am curious how would this behave if we added `initialRoute` as requested in #767

Reply:
> I am curious how would this behave if we added `initialRoute` as requested in #767

It should certainly be possible to get working. If you have `defaultUseCase` in `Widgetbook`, then have something like 

```dart
router = AppRouter(
   initialRoute: Uri.base.fragment.isNotEmpty
    ? Uri.base.fragment
    : routeFor(defaultUseCase), // or whatever
)
```